app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: 'blogpassreset@gmail.com'
```

Comment:
It isn't a good idea to include emails directly in the code, it's better to bring the data from the `.env`

Reply:
I'll make the change.